backend/api/Services/AreaService.cs

```diff
@@ -70,11 +70,13 @@ public async Task<IEnumerable<Area>> ReadAll()
             var installation = await installationService.ReadByName(installationCode);
             if (installation == null) { return null; }
 
-            return await context.Areas.Where(a =>
-                    a.Installation != null && a.Installation.Id.Equals(installation.Id) &&
-                    a.Name.ToLower().Equals(areaName.ToLower())
-                ).Include(a => a.SafePositions).Include(a => a.Installation)
-                .Include(a => a.Plant).Include(a => a.Deck).FirstOrDefaultAsync();
+            return await context.Areas.Where(a => a.Installation.Id.Equals(installation.Id) && a.Name.ToLower().Equals(areaName.ToLower()))
```

Comment:
This should use GetAreas() since this currently ignores granular access.

Reply:
Fixed, and also fixed for other functions in the same service.